src/main/resources/init.sh

```diff
@@ -77,11 +77,11 @@ status)
                 exit 0
             else
                 printf "%s\n" "Warning, Pid $PID appears to not correspond to service $SERVICE"
-                exit 0
             fi
         fi
-        printf "%s\n" "Process dead but pidfile exists"
-        exit 1
+        printf "%s\n" "Process dead but pidfile exists. Cleaning up."
+        rm -f $PIDFILE
+        exit 3
```

Comment:
(looks like the test condition is inverted too: `java.lang.AssertionError: Expected command 'dist/foo/service/bin/init.sh status' to be successful expected:<3> but was:<0>` - expected and actual seem to be swapped)